api/task_check_run.go

```diff
@@ -190,3 +191,64 @@ type TaskCheckRunStatusPatch struct {
 	Code   common.Code
 	Result string
 }
+
+// ConvertToErrorCode will convert advisor code into common code
```

Comment:
We can do it for now, but I think we can make the task check code to use advisor.Code directly with a namespace (namespace=advisor, code=advisor.Code.XXX).

Reply:
yeah, I think namespace is a great idea as it allows us to include future 3rd party modules as well.

Reply:
Let's do this in a separate PR.